MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java

```diff
@@ -891,14 +885,14 @@ private void maybeShowPopup(MouseEvent e) {
                             }
                         }
                     }
-                    if (techsFound > 0) {
-                        JMenuHelpers.addMenuIfNonEmpty(menu, menuElite, 20);
-                        JMenuHelpers.addMenuIfNonEmpty(menu, menuVeteran, 20);
-                        JMenuHelpers.addMenuIfNonEmpty(menu, menuRegular, 20);
-                        JMenuHelpers.addMenuIfNonEmpty(menu, menuGreen, 20);
-                        JMenuHelpers.addMenuIfNonEmpty(menu, menuUltraGreen, 20);
 
-                        popup.add(menu);
+                    if (techsFound > 0) {
+                        JMenuHelpers.addMenuIfNonEmpty(menu, menuElite);
```

Comment:
Why did the 20 get removed?

Reply:
Because I overrode the method it is calling to make 20 a default value. Better to use that instead of a magic number

Reply:
Got it.